fintech-examples/fintech-ui/src/app/bank/payment/payment-initiate/initiate.component.html

```diff
@@ -1,11 +1,28 @@
 <div class="card card-frame--big">
-  <div class="lacc-parent">
+  <div class="card-body">
```

Comment:
Are Fintech-examples related changes meant to be in this pull-request?

Reply:
No!  I created another branch for that. Something went wrong while I managed my branches but it's also ok I think